package.json

```diff
@@ -43,8 +43,8 @@
   "homepage": "https://github.com/aviklai/react-leaflet-google-layer#readme",
   "peerDependencies": {
     "leaflet": "^1.5.0",
-    "react": "^16.8.0",
-    "react-dom": "^16.8.0",
+    "react": "^17.0.1",
```

Comment:
I think it's better to do:
"react": ">=16.8.0"
for all the changes packages.
so it will support both react 16 and 17.

Reply:
I agree and add commit. please check it! https://github.com/aviklai/react-leaflet-google-layer/pull/20/commits/f308d84a2a00da3fd3fe6cd90fdc942391cfbc67